Classes/Editor/MovableViews/MovableViewCanvas.swift

```diff
@@ -130,9 +170,12 @@ final class MovableViewCanvas: IgnoreTouchesView, UIGestureRecognizerDelegate, M
     ///  - transformations: transformations for the view
     ///  - location: location of the view before transformations
     ///  - size: size of the view
-    func addView(view: MovableViewInnerElement, transformations: ViewTransformations, location: CGPoint, size: CGSize) {
+    func addView(view: MovableViewInnerElement, transformations: ViewTransformations, location: CGPoint, origin: CGPoint? = nil, size: CGSize, animated: Bool) {
```

Comment:
When recreating from an archive we don't want the animations, so this `animated` parameter is used.

Reply:
Do you think it can be `false` as default value then?